config/webpack/webpack.dev.js

```diff
@@ -27,6 +27,7 @@ module.exports = (env = {}) => {
         devServer: {
             contentBase: path.join(__dirname, '../../dist'),
             hot: true,
+            host: '0.0.0.0',
```

Comment:
Remove this, it doesn’t need to be committed!

You can pass `host` to the `webpack` by just adding  `--host 0.0.0.0` to the` yarn web` command.